src/lib/wp-cli-process.ts

```diff
@@ -11,7 +11,7 @@ export type MessageName = 'execute';
 export type WpCliResult = ReturnType< typeof executeWPCli >;
 export type MessageCanceled = { error: Error; canceled: boolean };
 
-const DEFAULT_RESPONSE_TIMEOUT = 120000;
+const DEFAULT_RESPONSE_TIMEOUT = 300 * 1000;
```

Comment:
While increasing the general timeout for every WP-CLI command. What do you think about increasing it even more for the specific import/export commands? Not a strong opinion.

We could pass the timeout as an option in `executeWpCliCommand`

https://github.com/Automattic/studio/blob/2c27eaf3b26045eca0545ca577b3a72b56544636/src/site-server.ts#L171

And then pass the value in the import
https://github.com/Automattic/studio/blob/2c27eaf3b26045eca0545ca577b3a72b56544636/src/lib/import-export/import/importers/importer.ts#L60

and export:

https://github.com/Automattic/studio/blob/2c27eaf3b26045eca0545ca577b3a72b56544636/src/lib/import-export/export/export-database.ts#L21

Reply:
Thanks for the review and suggestions!

I think we can leave as is for now and if we run into more issues with larger databases, we could refine the process further.